src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationUtilsTest.java

```diff
@@ -594,6 +594,24 @@ public void whenGettingLatestHearing_shouldReturnNullIfNoHearings() {
         assertNull(hearing);
     }
 
+    @Test
+    public void whenJointPartySubscriptionIsAdded() {
+        Subscription subscription = Subscription.builder().subscribeSms(YES).subscribeEmail(YES).wantSmsNotifications(YES).build();
+        SscsCaseDataWrapper wrapper = buildJointPartyWrapper(subscription, null).getSscsCaseDataWrapper();
+
+        SscsCaseData oldSscsCaseData = buildBaseWrapper(null, null).getOldSscsCaseData();
+
+        assertTrue(NotificationUtils.isJointPartySubscriptionCreate(wrapper.getNewSscsCaseData(), oldSscsCaseData));
+    }
+
+    @Test
+    public void whenJointPartySubscriptionIsAlreadyThere() {
```

Comment:
same here, the method should describe the test.
 perhaps;
shouldReturnFalse_whenThereIsAlreadyAJointSubscription